src/app/clusters/bindings/BindingManager.h

```diff
@@ -93,6 +96,11 @@ class BindingManager
      */
     CHIP_ERROR NotifyBoundClusterChanged(EndpointId endpoint, ClusterId cluster, void * context);
 
+    /*
+     * Notify the BoundDeviceChangedHandler that a binding has changed.
```

Comment:
Changed in what way?  This is not notifying removal of bindings, for example...

In general, what is the use case for this notification?  Right now it's not being used for anything, which makes it hard to see whether it makes sense at all.

Probably better to remove this notification for now, until it's actually being used.

Reply:
Updated name to reflect usage. Added TODO in tv-casting-app indicating how this will be used in the next PR.